dist_tplink_v3/scripts/misc-daemon

```diff
@@ -0,0 +1,102 @@
+#!/bin/sh
```

Comment:
This script is actulaly mostly artifacts from the Orbic which probably aren't even present or necessary for the TP link. I would ask that we find a cleaner way to start the daemon on TP Link. Either make our own init scirpt or add the rayhunter daemon start to a differnet init script that the TP link uses. As is this will almost certainly generate a bunch of errors at best.

Reply:
Actually if we are starting rayhunter in teh lighttpd script than we don't need this at all.